code/game/objects/items/devices/PDA/virus_cart.dm

```diff
@@ -68,7 +68,7 @@
 		if(target.cartridge)
 			difficulty += BitCount(target.cartridge.access&(CART_MEDICAL | CART_SECURITY | CART_ENGINE | CART_CLOWN | CART_JANITOR | CART_MANIFEST))
 			if(target.cartridge.access & CART_MANIFEST)
-				difficulty++ //if cartridge has manifest access it has extra snowflake difficulty
+				difficulty += 4 //if cartridge has manifest access it has extra snowflake difficulty
```

Comment:
this is a balance change, youre giving them more difficulty
just remove the else so they get 3

Reply:
That's what I thought initially, but it seems the writer of this code wanted manifest access to have a +4, one +2 from the part that adds it to manifest cartridges and the other +2 from the "else", (before the part I changed there is another "if" that checks the target PDA for all the different types of cartridges, including manifest. From what I can tell it was intended to add +2 to manifest then another +2 to all cartridges, including another +2 to manifest) and wanted the other cartridges to have a +2, with cartridgeless PDAs to have a 0. If I just remove the else, it would be a balance change for the other cartridges.

Since the "else" only applies to PDAs with cartridges, and not PDAs with none, removing the "else difficulty += 2" would be a rebalance.

Reply:
wdym one +2, thats a ++, which is += 1
i said to remove the else, not the else difficulty += 2

Reply:
Oh, sorry, I misunderstood. Fixing that now